src/syrupy/__init__.py

```diff
@@ -38,6 +37,13 @@ def pytest_addoption(parser: Any) -> None:
         dest="warn_unused_snapshots",
         help="Do not fail on unused snapshots",
     )
+    group.addoption(
+        "--snapshot-default-plugin",
```

Comment:
Wording should be `--snapshot-default-extension` since that's the language used elsewhere